extras/userdoc/md/documentation/scheduling-and-simulation-flow.md

```diff
@@ -77,9 +80,17 @@ wide without need leads to decreased performance due to more update calls and
 communication cycles (small *d<sub>min</sub>*), or increased memory consumption
 of NEST (large *d<sub>max</sub>*).
 
-Related topics
---------------
+## Spike generation and precision
+
+A neuron fires a spike when the membrane potential at the end of an update interval of
+the simulation resolution is above threshold. For most models, the membrane potential
+is then reset to some fixed value and clamped to that value during the refractory time.
+This means that the last membrane potential value at the last time step before the spike
+can vary, while the potential right after the step will usually be the reset potential
+(some models may deviate from this). The time of the spike is always the time at the end
```

Comment:
Add: "This also means that the membrane potential recording will never show values above the threshold."